modules/olm-default-install-behavior.adoc

```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// * operators/understanding/olm-multitenancy.adoc
+
+:_content-type: CONCEPT
+[id="olm-default-install-modes-behavior_{context}"]
+= Default Operator install modes and behavior
+
+When installing Operators with the web console as an administrator, you typically have two choices for the install mode, depending on the Operator's capabilities:
+
+Single namespace:: Installs the Operator in the chosen single namespace, and makes all permissions that the Operator requests available in that namespace.
+
+All namespaces:: Installs the Operator in the default `openshift-operators` namespace to watch and be made available to all namespaces in the cluster.Makes all permissions that the Operator requests available in all namespaces.
```

Comment:
In some cases, an operator can define metadata to give the user a second option for that operator's suggested namespace